nextgenstar/engine
# [Fix] Move files & fix locations

### Description of the Change

The general project related guides and git files should not go under docs folder. All such files should be present in the project root.

### Analysis and Design

N/A

### Implementation Details

All general guides are moved to project root.

### Impacts

Breaking change: No

Other impacts: No


### Verification

- Check files are in project root.
- Check all links are functional in the documents.
